composer.json

```diff
@@ -7,7 +7,7 @@
         "php": ">=7.4",
         "ext-json": "*",
         "composer/package-versions-deprecated": "^1.11",
-        "doctrine/doctrine-bundle": "2.2.4",
+        "doctrine/doctrine-bundle": "2.3.0",
```

Comment:
why keeping an exact version now ? Are you still bisecting ?

Reply:
I just wanted to confirm that this was a fix. I'll follow this up with a second pr to allow the latest versions of all doctrine packages.